bundle/manifests/odf-operator.clusterserviceversion.yaml

```diff
@@ -198,6 +198,8 @@ spec:
                 - --health-probe-bind-address=:8081
                 - --metrics-bind-address=127.0.0.1:8080
                 - --leader-elect
+                - --console-image=badhikar/odf-console:latest
```

Comment:
This should also go in the config/manager/manager.yaml But I think we should have this as an ENV variable.